packages/backend/src/ee/database/migrations/20250703123404_ai_agent_group_access.ts

```diff
@@ -0,0 +1,40 @@
+import { Knex } from 'knex';
+
+const AiAgentGroupAccessTableName = 'ai_agent_group_access';
+const AiAgentsTableName = 'ai_agent';
+const GroupsTableName = 'groups';
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable(AiAgentGroupAccessTableName))) {
+        await knex.schema.createTable(
+            AiAgentGroupAccessTableName,
+            (tableBuilder) => {
+                tableBuilder
+                    .uuid('group_uuid')
+                    .notNullable()
+                    .references('group_uuid')
+                    .inTable(GroupsTableName)
+                    .onDelete('CASCADE');
+                tableBuilder
+                    .uuid('ai_agent_uuid')
+                    .notNullable()
+                    .references('ai_agent_uuid')
+                    .inTable(AiAgentsTableName)
+                    .onDelete('CASCADE');
+                tableBuilder
+                    .timestamp('created_at', { useTz: false })
+                    .notNullable()
+                    .defaultTo(knex.fn.now());
+                tableBuilder
+                    .timestamp('updated_at', { useTz: false })
+                    .notNullable()
+                    .defaultTo(knex.fn.now());
```

Comment:
this seems redundant, we only ever create/delete the row on this table

Reply:
makes super sense, will remove `updated_at`, update types and merge